src/main/java/org/jabref/model/openoffice/uno/UnoBookmark.java

```diff
@@ -52,16 +52,14 @@ public static Optional<XTextRange> getAnchor(XTextDocument doc, String name)
      * <p>
      * In LibreOffice the another name is in "{name}{number}" format.
      *
-     * @param name   For the bookmark.
-     * @param range  Cursor marking the location or range for the bookmark.
-     * @param absorb Shall we incorporate range?
      * @return The XNamed interface of the bookmark.
      * result.getName() should be checked by the caller, because its name may differ from the one requested.
+     * @param documentAnnotation The DocumentAnnotation object
```

Comment:
Can be removed, because the variable name also says that.

(Moreover, `@param` should have been put above `@return`. Our checkstyle seems to miss that configuration :p)